src/eye_extractor/dr/hemorrhage_type.py

```diff
@@ -17,59 +18,85 @@ class HemorrhageType(enum.IntEnum):
 
 INTRARETINAL_PAT = re.compile(
         r'\b('
-        r'intraretinal\s*hemorrhage'
-        r'|hemorrhage\s*intraretinal'
+        r'intraretinal\s*hem(orrhage|e)'

```

Comment:
Might be worth refactoring these to use f-strings. E.g., `heme = r'hem(orrhage|e)'`, then `rf'dot blot\s*{heme}'`. That way, any future modifications only need to be made in a single place.